src/components/Strategies/StrategyDetails.tsx

```diff
@@ -71,6 +73,10 @@ function StrategyDetails() {
     });
   }, [strategy?.blocks]);
 
+  const handleModelCancel = () => {
```

Comment:
I think you meant `Modal`? also, you need to call the `cancel` function here (see cancel in `handleSubmit`)